src/python/py/models/builder.py

```diff
@@ -2799,7 +2963,16 @@ def create_model(model_name, input_path, output_dir, precision, execution_provid
     extra_kwargs = {} if os.path.isdir(input_path) else {"cache_dir": cache_dir}
     hf_name = input_path if os.path.isdir(input_path) else model_name
     hf_token = parse_hf_token(extra_options.get("hf_token", "true"))
-    config = AutoConfig.from_pretrained(hf_name, token=hf_token, trust_remote_code=True, **extra_kwargs)
+
+    is_peft = os.path.isdir(adapter_path) and "adapter_config.json" in os.listdir(adapter_path)
+    peft_model_name = adapter_path if is_peft else None
+    peft_config = None
+    if is_peft:
+        from peft import PeftConfig
+        peft_config = PeftConfig.from_pretrained(peft_model_name, use_auth_token=True, trust_remote_code=True, **extra_kwargs)
```

Comment:
use_auth_token was replaced with `token=hf_token` in the deleted line 2802

Reply:
Yeah I think I did not do the rebase correctly